pkg/codegen/python/gen.go

```diff
@@ -723,7 +723,7 @@ func (mod *modContext) genFunction(fun *schema.Function) (string, error) {
 	}
 
 	// Copy the function arguments into a dictionary.
-	fmt.Fprintf(w, "    __args__ = dict()\n\n")
+	fmt.Fprintf(w, "    __args__ = dict()\n")
 	fmt.Fprintf(w, "\n")
```

Comment:
Any idea why we have this extra line before setting the values in `__args___`? Wondering if we could get rid of this too.

Reply:
Yeah there's no apparent reason for it. Will remove.